applications/convection_diffusion_application/tests/run_cpp_unit_tests.py

```diff
@@ -0,0 +1,9 @@
+from KratosMultiphysics import *
```

Comment:
is this file needed?

Reply:
Well, eventually if some cpp test is added yes

Reply:
I tend to agree with @philbucher, I'm not sure we need to add a cpp test runner in every application.

Maybe we could add the two lines of the run() function to the main test_ConvectionDiffusionApplication file?